stock_whole_kit_constraint/README.rst

```diff
@@ -0,0 +1,97 @@
+==========================
+Stock whole kit constraint
+==========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fmanufacture-lightgray.png?logo=github
+    :target: https://github.com/OCA/manufacture/tree/12.0/stock_whole_kit_constraint
+    :alt: OCA/manufacture
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/manufacture-12-0/manufacture-12-0-stock_whole_kit_constraint
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/129/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to configure a product kit to disallow partial deliveries so the
+componentes can't be partially delivered.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To disallow kits partial deliveries:
+
+#. Go to the kit product template and then to the *Inventory* tab, *Logistics* group.
+#. Set *Allow Partial Kit* off.
```

Comment:
Isn't the default as False?

Reply:
The default is to allow partial deliveries

Reply:
Yes, so the redaction should change.

Reply:
What do you mean?

Reply:
You have to put something similar to:
```
To allow/disallow the partial delivery of kits:

#. Go to...
#. The check "Allow Partial Kit" controls this. If marked, it will allow it.
#. By default, the check is not marked.

Reply:
Ok, changed